template/pnpm-lock.yaml

```diff
@@ -0,0 +1,5862 @@
+lockfileVersion: '9.0'
```

Comment:
Does it make sense to check in the lock file? I would imagine this doesn't do anything if you pick a different package manager anyway?